Tools-Override/FrameworkTargeting.targets

```diff
@@ -215,4 +215,114 @@
     </ItemGroup>
   </Target>
 
+  <!-- Binplacing targets -->
+  <Target Name="BinPlace"
+          DependsOnTargets="GetBinPlaceDirs;BinPlaceFiles;BinPlaceProps"
+          AfterTargets="CopyFilesToOutputDirectory"
+          Condition="'$(BinPlaceRef)' == 'true' OR '$(BinPlaceRuntime)' == 'true'" />
```

Comment:
Yeah, this seems wrong. How would we envision other repos "opting in" to this binplacing logic? I guess they would set some `@(BinPlaceConfigurations)` if they wanted this to start working.